site/content/features/gatherers.html

```diff
@@ -0,0 +1,394 @@
+---
+title: Stream Gatherers (JEP 485)
+copyright: Cay S. Horstmann 2024. All rights reserved.
+jep: 485
+jdkversion: 23
+---
+    <script src='https://javaalmanac.io/app/sandbox-bundle.js'></script> 
+    <script>//<![CDATA[
```

Comment:
Anything is possible, but this is less work for me. And it works now, with your `params: sandbox: true`, right? What I'd really like to be able to do is to put the script into a file, but I don't know where to put it so that Hugo will include it in the build.

Reply:
Exactly. With the new `sandbox` frontmatter embedding of the sandbox is enforced.

I'll try to extract the converter script for re-use. How should the script be called? `convertPreToSandbox`?

Reply:
convertSandboxDivToSandbox.js

Reply:
I also noted, that the script had-codes Java 23. To make it a bit more flexible I would activate the script with the following frontmatter:

    params:
      sandbox: true
      convertSandboxDivToSandbox: java23

This will allow to specify different Java versions.

Reply:
I just pushed a proposal how the script can be extracted. Is this what you where looking for?

Reply:
I just ran it and somehow it skips every second sandbox. I'll look into it tonight.

Reply:
I know what it is: I moved the removal of the div elements into the first loop. Now I know why you had an extra loop. Fixed now

Reply:
Ok, looks good, thanks!. I had a look at site/layouts/partials/convertsandboxdivtosandboxscript.html. Now that I see how it fits into Hugo, I can tweak it if I need to.  

BTW, I tried debugging the script, and got nowhere because I thought the script was included with the bundled minified JS. But now I see it at the end of the page.

That's it, right? Ready to squash and merge?

Reply:
I decided to inline it for two reasons:

1) It is rather small and may not justify another HTTP request
2) I want to execute it synchronously so that the elements are replaced before Vue starts